java/src/apps/Apps.java

```diff
@@ -287,7 +283,11 @@ public boolean execute() {
         // populate GUI
         log.debug("Start UI");
         setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
-
+        
+        // Prepare font lists
```

Comment:
This needs to be moved back to its prior location because:
- the UserPreferencesManager relies on the font list being completed
- you have the earlier "press F8 to skip logix" and profile selector windows when JMRI is doing nothing but giving a user a window of time in which to respond. Triggering this reading of the font lists after that wastes that opportunity only to hold up the user later, when (for example) loading a panel (since, as far as I can tell, every panel class has a menu item or combo box associated with it that includes the font lists.

Reply:
Actually the font loading task is asynchronous and may, or not, be complete before the UserPrefrencesManager requires it. My testing showed it's not a problem to do it later.

Anyway, as suggested yesterday I'm working on removing this thing.